pom.xml

```diff
@@ -11,7 +11,7 @@
 	<groupId>org.dcsa</groupId>
 	<artifactId>dcsa_tnt</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
-	<packaging>war</packaging>
+	<packaging>jar</packaging>
```

Comment:
I am not sure I understand this change.  I would expect TNT to be a war file - though in case this change is correct (it might be), then it would imply an update to the Dockerfile which still looks for a war file.

Reply:
We should be using jar as far as I know, that's what @HenrikHL had mentioned too. I updated the docker file too btw.